apps/mobile/src/services/analytics/types.ts

```diff
@@ -56,3 +56,51 @@ export type AnalyticsLabel =
   | (typeof ANALYTICS_LABELS.TRANSFER_TYPES)[TransferInfoType]
   | (typeof ANALYTICS_LABELS.SETTINGS_TYPES)[SettingsInfoType]
   | (typeof ANALYTICS_LABELS.ENHANCED)[keyof typeof ANALYTICS_LABELS.ENHANCED]
+
+/**
+ * MixPanel User Attributes for Safe Wallet (Mobile)
+ *
+ * These attributes are used for cohort analysis and user segmentation
+ * Shared with web app for consistency
+ */
+export interface SafeUserAttributes {
+  /** On-chain address of the Safe (Primary key for cohort analysis) */
+  safe_id: string
+
+  /** Timestamp when the Safe was deployed (Life of user) */
+  created_at: Date
+
+  /** Each version has different properties (Filter out older Safes) */
+  safe_version: string
+
+  /** Total number of wallet addresses/signers on this Safe (Engagement metric) */
+  num_signers: number
+
+  /** Number of signatures required to execute a transaction (Setup intent) */
+  threshold: number
+
+  /** Blockchain networks (Context for segmentation) */
+  networks: string[]
+
+  /** Timestamp of most recent transaction (Churn alerting) */
+  last_tx_at: Date | null
+
+  /** ID of parent Space if the Safe is grouped in a Space (Space-level grouping) */
```

Comment:
IMO these comments are more confusing than they help. I would suggest removing them to increase readability of the file.